docs/API/Users_Guide/Data_Analysis.md

```diff
@@ -150,6 +153,379 @@ gene_start      gene_end        symbol  id
+
+```Filter
+{
+  "case_set_id": "gOq-6bhPHBg7b-f7dy2nPw",
+  "gene_set_id": "mhiW6Y8PFLqNo7u9qXCtNA"
+}
+```
+
+```Shell
+curl -X 'POST' \
+  'https://api.gdc.cancer.gov/gene_expression/availability' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "case_set_id": "gOq-6bhPHBg7b-f7dy2nPw",
+  "gene_set_id": "mhiW6Y8PFLqNo7u9qXCtNA"
+}'
+```
+
+```Response
+{
+    "cases": {
+        "details": [
+            {
+                "case_id": "6d4f38db-a97b-4dc0-8dc5-2ac7f2cc5e38",
+                "has_gene_expression_values": false
+            },
+            {
+                "case_id": "e3b32485-b204-43a7-93a5-601408fcdf96",
+                "has_gene_expression_values": true
+            }
+        ],
+        "with_gene_expression_count": 1,
+        "without_gene_expression_count": 1
+    },
+    "genes": {
+        "details": [
+            {
+                "gene_id": "ENSG00000141510",
+                "has_gene_expression_values": true
+            },
+            {
+                "gene_id": "ENSG00000181143",
+                "has_gene_expression_values": true
+            }
+        ],
+        "with_gene_expression_count": 2,
+        "without_gene_expression_count": 0
+    }
+}
+```
+
+__Example 2__: A user wants to get the availability of gene expression data for both case ids and gene ids.
+
+```Filter
+{
+  "case_ids": [
+    "6d4f38db-a97b-4dc0-8dc5-2ac7f2cc5e38",
+    "e3b32485-b204-43a7-93a5-601408fcdf96"
+  ],
+  "gene_ids": [
+    "ENSG00000141510",
+    "ENSG00000181143"
+  ]
+}
+```
+
+```Shell
+curl -X 'POST' \
+  'https://api.gdc.cancer.gov/gene_expression/availability' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "case_ids": [
+    "6d4f38db-a97b-4dc0-8dc5-2ac7f2cc5e38",
+    "e3b32485-b204-43a7-93a5-601408fcdf96"
+  ],
+  "gene_ids": [
+    "ENSG00000141510",
+    "ENSG00000181143"
+  ]
+}'
+```
+
+```Response
+{
+    "cases": {
+        "details": [
+            {
+                "case_id": "6d4f38db-a97b-4dc0-8dc5-2ac7f2cc5e38",
+                "has_gene_expression_values": false
+            },
+            {
+                "case_id": "e3b32485-b204-43a7-93a5-601408fcdf96",
+                "has_gene_expression_values": true
+            }
+        ],
+        "with_gene_expression_count": 1,
+        "without_gene_expression_count": 1
+    },
+    "genes": {
+        "details": [
+            {
+                "gene_id": "ENSG00000141510",
+                "has_gene_expression_values": true
+            },
+            {
+                "gene_id": "ENSG00000181143",
+                "has_gene_expression_values": true
+            }
+        ],
+        "with_gene_expression_count": 2,
+        "without_gene_expression_count": 0
+    }
+}
+```
+
+### Gene Expression Values Endpoint
+
+Get gene expression values for the given cases and genes. The response is a tsv containing the expression values for genes to cases.
```

Comment:
"tsv" is normally capitalized. not sure if that's the situation in our docs though.

Reply:
If I had anything to do with this, it's probably not standardized.  Capital is cool though.

Reply:
changed :)